packages/core/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@stamp/core",
+  "version": "0.0.1",
+  "description": "Core functions for creating stamps",
+  "repository": "https://github.com/stampit-org/stamp/packages/core",
```

Comment:
Typo in the URL. Take a look at the `babel-core` repository field:
```
  "repository": "https://github.com/babel/babel/tree/master/packages/babel-core",
```
Same typo in other packages.